### ISSUE TITLE
orderByPopular doesn't order correctly

### ISSUE THREAD

**Issue by mijewe** [24hoursmedia-craftcms/views-work#107]
I'm ordering by entries using `.orderByPopular('total', 0)` as per the docs, but the order doesn't appear to be by most popular.

```
    {% set results = craft.entries.orderByPopular('total', 0).all() %}

    {% for result in results %}
        <li>{{ result.title }} ({{ result.viewsWork.total }})</li>
    {% endfor %}
```

returns (for example)

```
entry one (8)
entry two (12)
entry three (0)
entry four (3)
```

I'm using:
Craft 4.5.6
MariaDB 10.11.4
Views Work 2.1.2

**Comment by davorpeic** [comment]
@mijewe Aren't you missing section?

**Comment by mijewe** [comment]
I was using the most minimal example I could to illustrate the point.

With or without a `section` parameter, it should still be returning results in order of most popular

**Comment by mijewe** [comment]
Hi :)

any update on this? This feels like a pretty fundamental aspect of the plugin.

Or I'm missing something

**Comment by 24hoursmedia-craftcms** [comment]
Hi :) Sorry it is kind of busy with Black Friday.

I am currently looking into it, I can reproduce the issue and come back to it soon!

**Comment by 24hoursmedia-craftcms** [comment]
Thank you for the report, the issues was introduced in another release with multisite support and escaped testing.
It should be fixed in 2.1.3 (just released).
